zerver/openapi/python_examples.py

```diff
@@ -1081,6 +1081,7 @@ def set_typing_status(client: Client) -> None:
         "to": [user_id1, user_id2],
     }
     result = client.set_typing_status(request)
+
```

Comment:
Added this space(and below) intentionally to make it look good in API docs with each of start, stop separated as a different chunk.